### ISSUE TITLE
Wall post button don't post on first click.

### ISSUE THREAD

**Issue by sashkooo** [kavolorn/VK-ANE#4]
Wall post button don't post on first click, only ask permissions. 
Also get error in output:
[trace] error {"errorCode":-101,"vkErrorCode":5}

On next clicks it works fine.

**Comment by kavolorn** [comment]
There is an error handler in the code that should handle this situation:

```actionscript
private function errorHandler(error:Object, request:Object)
{
	trace('error', JSON.stringify(error));

	if (error.errorCode == VKError.VK_API_ERROR && error.vkErrorCode == 5)
	{
		_statusBar.text = "User is logged out.";
		VK.getInstance().authorize(SCOPE, true, false, false);
	}
}
```

I never saw that it is not working. If authorization is required then operation is canceled and after authorization you have to click once again. Do you see this behaviour?

**Comment by sashkooo** [comment]
Yes, I see, I just report that wall post button in the demo not working as well, it have do post after log in.

**Comment by kavolorn** [comment]
Not a bug, but could be better :)
